src/account_multisig.ts

```diff
@@ -43,11 +43,16 @@ export class AccountMultisig extends Account {
         this.contract = <MultisigContract>getContract(this);
     }
 
-    async addKey(publicKey: string | PublicKey, contractId?: string, methodName?: string, amount?: BN): Promise<FinalExecutionOutcome> {
-        if (contractId) {
-            return super.addKey(publicKey, contractId, MULTISIG_CHANGE_METHODS.join(), MULTISIG_ALLOWANCE)
+    /** 
+     * If contractId matches given accountId, mutlisig only address another multisig access key.
```

Comment:
@ilblackdragon not sure I understand the intention here. This seems to break everything, e.g. you cannot login to apps anymore. AFAIK this is perfectly possible with multisig contract (I just tried logging in into guest-book today).

Also shouldn't it even be possible to add another full access key (e.g. for seed phrase recovery)?